Source/AliveLibAE/Scrab.cpp

```diff
@@ -1246,15 +1246,18 @@ __int16 Scrab::AI_ChasingEnemy_1_4A6470()
         field_14C_spotting_abe_timer = sGnFrame_5C1B84 + field_148_attack_duration;
     }
 
-    if (Event_Is_Event_In_Range_422C30(kEventAbeOhm, field_B8_xpos, field_BC_ypos, -1) && field_11C_sub_state != 26)
+    if (Event_Is_Event_In_Range_422C30(kEventAbeOhm, field_B8_xpos, field_BC_ypos, -1) && field_11C_sub_state != 26) //TODO OG bug? field_108_next_motion instead of field_11C_sub_state
     {
         field_108_next_motion = eScrabMotions::M_HowlBegin_26_4A9DA0;
         return AI_ChasingEnemy::eState1_Panic_4;
     }
 
-    LiftPoint* pLiftPoint = static_cast<LiftPoint*>(pObj);
-    if (pLiftPoint && pLiftPoint->field_4_typeId == Types::eLiftPoint_78 && 
-        !pLiftPoint->vOnAnyFloor_461920() && field_11C_sub_state != AI_ChasingEnemy::eState1_Panic_4)
+    LiftPoint* pLiftPoint = static_cast<LiftPoint*>(sObjectIds_5C1B70.Find_449CF0(field_110_id));
+    if (!pLiftPoint || pLiftPoint->field_4_typeId != Types::eLiftPoint_78)
+    {
+        pLiftPoint = nullptr;
```

Comment:
comment that this is an OG fix for pLiftPoint being used later even if it isn't a lift point